README.md

```diff
@@ -33,7 +33,7 @@ or join us in the **#nautobot** Slack channel on [Network to Code](https://netwo
 
 ## Installation
 
-Please see [the documentation](https://nautobot.readthedocs.io/en/latest/) for
+Please see [the documentation](https://nautobot.readthedocs.io/) for
```

Comment:
This should link to https://nautobot.readthedocs.io/en/latest/installation/

Reply:
fixed.